clusterloader2/pkg/modifier/modifier.go

```diff
@@ -60,3 +75,69 @@ func (m *simpleModifier) ChangeTest(c *api.Config) {
 		}
 	}
 }
+
+func (m *simpleModifier) modifyOverwrite(c *api.Config) error {
+	for _, o := range m.overwriteTestConfig {
+		kv := strings.Split(o, "=")
+		if len(kv) != 2 {
+			errMsg := fmt.Sprintf("Not a key=value pair: %s", o)
+			klog.Error(errMsg)
```

Comment:
don't log errors if you return them - it's responsibility of the calls to do whether they want with it

Reply:
Same everywhere below